content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,319 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+<abbr>Experiments</abbr> proliferate quickly in ML projects where there are many
+parameters to tune or other permutations of the code. We can organize such
+projects and keep only what we ultimately need with `dvc experiments`. DVC can
+track experiments for you so there's no need to commit each one to Git. This way
```

Comment:
New users that come here won't understand this part `to commit each one to Git.`. Committing experiments to Git is very DVC-specific.

Reply:
Agreed. This is also an issue below when introducing _ephemeral_ experiments. Need to think of how to introduce experiments without assuming user knows about dvc's gitops workflow.

Reply:
I've updated the intro in 4f56d8.